plugins/turing/go.mod

```diff
@@ -125,12 +125,9 @@ require (
 )
 
 replace (
-	// To remove once the next version of Turing (v1.9.0) has been cut and released
-	github.com/caraml-dev/turing/engines/experiment => github.com/caraml-dev/turing/engines/experiment v0.0.0-20230104094236-477ac2392c59
-	github.com/caraml-dev/turing/engines/router => github.com/caraml-dev/turing/engines/router v0.0.0-20230104094236-477ac2392c59
+	github.com/caraml-dev/turing/engines/experiment => github.com/caraml-dev/turing/engines/experiment v0.0.0-20230414061300-682496141466
```

Comment:
Could you also mention the update to the Turing dependency in the PR description? This is needed purely for the update to the MLP dependency (which is why the replace directive for MLP below can be removed), right?

Reply:
Yes, that's my understanding as well!